pages/api/app.js

```diff
@@ -37,7 +38,33 @@ function buildAppProfile(hostname, clientId) {
 }
 
 export default function handler(req, res) {
+  if (req.method !== "GET") {
+    return res.status(400).send("Bad Request");
+  }
+
   const clientId = `https://${req.headers.host}/api/app`;
   const hostname = `https://${req.headers.host}/`;
-  res.status(200).json(buildAppProfile(hostname, clientId));
+
+  const acceptedType = accepts(req).types([
+    "application/json",
```

Comment:
It isn't obvious that the first type here is returned in the absence of an `Accept` header (which I find a bit odd, as opposed to returning `false` like it does when the negotiation fails). I think it would be worth documenting in a comment.